Adamant/Modules/Settings/Notifications/Helpers/NotificationSoundsPickerView.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  NotificationSoundsPickerView.swift
+//  Adamant
+//
+//  Created by Yana Silosieva on 17.08.2024.
+//  Copyright © 2024 Adamant. All rights reserved.
+//
+
+import SwiftUI
+
+struct NotificationSoundsPickerView: UIViewControllerRepresentable {
```

Comment:
Let's use a different approach, I will show you on the call

Reply:
I deleted NotificationSoundsPickerView, it was unnecessary because I wrote the notification sounds screen in SwiftUI